searchlib/src/vespa/searchlib/diskindex/zcposocciterators.h

```diff
@@ -35,10 +36,11 @@ class ZcPosOccIterator : public ZcPostingIterator<bigEndian>
     using DecodeContext = std::conditional_t<dynamic_k, bitcompression::EGPosOccDecodeContextCooked<bigEndian>, bitcompression::EG2PosOccDecodeContextCooked<bigEndian>>;
     DecodeContext _decodeContextReal;
 public:
-    ZcPosOccIterator(Position start, uint64_t bitLength, uint32_t docIdLimit, bool decode_cheap_features,
-                      uint32_t minChunkDocs, const index::PostingListCounts &counts,
-                      const bitcompression::PosOccFieldsParams *fieldsParams,
-                      const fef::TermFieldMatchDataArray &matchData);
+    ZcPosOccIterator(Position start, uint64_t bitLength, uint32_t docIdLimit,
+                     bool decode_cheap_features, bool decode_normal_features,
```

Comment:
The order of the parameter list does not match the one in the cpp file. In the cpp file decode_normal_features comes before decode_cheap_features. We should have the same order here to avoid confusion.

Reply:
Fixed